examples/github.com-acidanthera-OpenCore/Makefile

```diff
@@ -1,6 +1,8 @@
 all:
+	make clean
 	mkdir build
-	cd build && ../build.sh
+	cd build && . ../build.sh
+	cd build && . ../build1.sh
```

Comment:
Let's call it `build_debug.sh` and `build_release.sh` here and below :)

Reply:
done